src/components/Shared/WindowToolbar.js

```diff
@@ -19,6 +19,11 @@ const WindowToolbar = ({ onSave, onClear, onInfo , disabledSubmit, editMode=fals
         Submit
         </Button>
       )}
+       {onApply && (
+        <Button onClick={onApply} variant='contained' disabled={disabledApply}>
```

Comment:
you are using onAppy and disabledApply without sending them as props

Reply:
fixed